sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -113,58 +115,58 @@ We define three new functions needed to perform the main and common operations o
 
 The base pointer determines the starting address of the matrix to be loaded/stored. `layout` determines whether the data are being read/written in a row (`row_major`), column major (`column_major`) fashion, or if the data has already been transformed into VNNI format (`packed_a`, `packed_b`). `stride` describes the number of elements between consecutive rows for row major and packed layout,  columns for column major layout. 
 
-Note that for getting maximum performance on AMX, prepacking data in the memory is necessary. If users did not specify the packed layouts (`packed_a` in column major case, `packed_b` in row major case), transforms done by the implementation will be slow due to extra scatter/gather operations. Hence, we expose these layouts `packed_a` and `packed_b` to the user to specify that A and/or B have already been VNNIed. The packed or VNNI layout is introduced in `VNNI layout` section below.
+Note that for getting maximum performance on AMX and DPAS, prepacking data in the memory is necessary. If users did not specify the packed layouts (`packed_a` in column major case, `packed_b` in row major case), transforms done by the implementation will be slow due to extra scatter/gather operations. Hence, we expose these layouts `packed_a` and `packed_b` to the user to specify that A and/or B have already been VNNIed. The packed or VNNI layout is introduced in `VNNI layout` section below.
```

Comment:
It is implied in various places that `packed_a` should only be used with matrix A and `packed_b` with matrix_b.  Here it states "`packed_a` in column major case, `packed_b` in row major case".  This gives the impression that matrix A must always be indexed column major and matrix B row_major for AMX and DPAS (at least when they are `packed`).  If what I wrote is correct it would be clearer to state this explicitly, or otherwise clarify the situation.